docs/CLI - Troubleshooting Guide .md

```diff
@@ -35,3 +35,6 @@ az account set --subscription <correct-subscription>
 
```

Comment:
This file name has changed, could you move the changes to new file "Troubleshooting Guide .md"